packages/rocketchat-apps/client/admin/appManage.js

```diff
@@ -0,0 +1,175 @@
+import _ from 'underscore';
+import s from 'underscore.string';
+
+Template.appManage.onCreated(function() {
+	const instance = this;
+	this.id = new ReactiveVar(FlowRouter.getParam('appId'));
+	this.ready = new ReactiveVar(false);
+	this.processingEnabled = new ReactiveVar(false);
+	this.app = new ReactiveVar({});
+	this.settings = new ReactiveVar({});
+
+	const id = this.id.get();
+
+	Promise.all([
+		RocketChat.API.get(`apps/${ id }`),
+		RocketChat.API.get(`apps/${ id }/settings`)
+	]).then((results) => {
+		instance.app.set(results[0].app);
+
+		Object.keys(results[1].settings).forEach((k) => {
+			results[1].settings[k].i18nPlaceholder = results[1].settings[k].i18nPlaceholder || ' ';
+			results[1].settings[k].value = results[1].settings[k].value || results[1].settings[k].packageValue;
+			results[1].settings[k].oldValue = results[1].settings[k].value;
+		});
+
+		instance.settings.set(results[1].settings);
+		this.ready.set(true);
+	}).catch(() => {
+		//TODO: error handling
```

Comment:
Can we just toss an error for now?  That way if this is hit during testing, we at least know where it was?